portfolio/src/main/webapp/style.css

```diff
@@ -1,16 +1,42 @@
 #content {
+  margin-top: 50px;
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.divider {
+  width: 100px;
+  height: auto;
+  display: inline-block;
 }
 
 body {
   margin: 0;
-  font-family: Arial, Helvetica, sans-serif;
+  font-family: Helvetica, Arial, sans-serif;
+}
+
+img {
+  opacity: 0.6;
+  transition: 0.3s;
+}
+
+img:hover {
+  opacity: 1;
+}
+
+img[src="images/profile.jpeg"] {
+  opacity: 1;
+}
+
+
+.image-link {
+  text-decoration: none;
+}
+
+
+.profile {
+  opacity: 1;
 }
```

Comment:
Instead of setting all img to have opacity 0.6 here, what if you do 
```
.image-link img {
  opacity: 0.6;
  transition: 0.3s;
}
```
And same with `img:hover` to 'image-link img:hover'

That way you don't have to set opacity back to 1 for your other images.